extensions/wikia/ArticleVideo/scripts/ooyala-player.js

```diff
@@ -42,6 +42,7 @@ define('ooyala-player', ['wikia.browserDetect'], function (browserDetect) {
 
 		messageBus.subscribe(window.OO.EVENTS.PLAYBACK_READY, 'ui-update', function () {
 			self.onPlaybackReady();
+			// messageBus.publish("replay", false);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3ACommentedCode)